src/plugins/data/public/public.api.md

```diff
@@ -368,6 +368,20 @@ export enum BUCKET_TYPES {
 // @public
 export const castOpenSearchToOsdFieldTypeName: (opensearchType: OPENSEARCH_FIELD_TYPES | string) => OSD_FIELD_TYPES;
 
+// @public
```

Comment:
Was this manually entered?

Reply:
yes, this is manually entered as i did not find a mechanism to auto generate the api doc. Are you aware of any mechanism that does that?